content/docs/guide/map_submission/fixing_entities.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Fixing Entities
+categories:
+  - guide
+tags:
+  - mapping
+  - guidelines
+weight: 4
+---
+
+# Introduction
+This guide provides a list of fixes to apply to entities when porting maps  
+If the entity you are looking for is not in here please ask for help in **#map-porting** channel on our [Discord](https://discord.gg/momentummod)  
+Detailed information on entities can also be found on [Valve Developer Wiki](https://developer.valvesoftware.com/wiki/Main_Page)
+
+# Boosters
+Boosting players can be achieved in various ways and therefore triggers need to be evaluated on a case-by-case basis.  
+This guide will teach you how to fix every common scenario
+
+## trigger_push and trigger_multiple
+trigger_push applies a constant boost to a player inside it's volume  
+trigger_multiple can be used for many purposes, however here we only care about it when it gives the player a singular burst of speed  
+Both need to be modified depending on how you activate them
+
+1. Open entity tools by typing `devui_show entitytools` in console
+    - You can bind this to a key for ease of access, `bind <key> "devui_show entitytools"` in console
+2. Open 'Boost Triggers' section and find the trigger you want to edit
+
+{{<hint info>}}
+
+**Entity tools** will list all boost triggers on the map. You can teleport to them by clicking 'Teleport to Trigger' and fix each one based on scenarios listed below  
+When making these changes it's perfectly fine to export them only once, when you are done with everything, however there is no downside to gradually saving your progress
+
+{{</hint>}}
+{{<hint info>}}
+
+You can change the font size of **Entity Tools** by using `devui_font_scale` command in console
+
+{{</hint>}}
+
+### Scenario 1: Constant push is required
+On some maps it is necessary for the player to be constantly pushed while inside of the trigger.  
+
+#### If the boost is applied in the air
+No modifications are required!
+
+#### If the boost is applied while walking on the ground
+3. Check 'Standable Surface Collision Filter'
+4. [Export to Lumper](/guide/map_submission/fixing_entities/#export-to-lumper)
+{{<hint warning>}}
+
+Only apply this fix if the walkable surface is completely flat  
+If it's not flat, don't modify the trigger in any way  
+
+{{</hint>}}
+![Floor boost](/images/map_porting/floor_boost.png)
```

Comment:
You might need to add a new line before this image. It's displaying next to the "Export to Lumper" text for me.